local-cli/runAndroid/runAndroid.js

```diff
@@ -56,10 +56,10 @@ function getAdbPath() {
 }
 
 // Runs ADB reverse tcp:8081 tcp:8081 to allow loading the jsbundle from the packager
-function tryRunAdbReverse(device) {
+function tryRunAdbReverse(args, device) {
```

Comment:
Instead of passing args through, can you pass `packagerPort`? That keeps the API saner.